lib/raven/integrations/rails/middleware/public_exceptions_catcher.rb

```diff
@@ -0,0 +1,22 @@
+module Raven
+  class Rails
+    module Middleware
+      module PublicExceptionsCatcher
+        def self.included(base)
+          base.send(:alias_method_chain, :call, :raven)
+        end
+
+        def call_with_raven(env)
+          request = ActionDispatch::Request.new(env)
+          status  = request.path_info[1..-1].to_i
+
+          if status >= 500
```

Comment:
Actually that's already checked in the `capture_exception` method itself. I think this is fine - the explanation for not sending non-5xx errors is that they've been handled gracefully by Rails' ShowExceptions logic. Not sure people want to hear about those.